plugins/mcreator-localization/lang/texts.properties

```diff
@@ -3,12 +3,24 @@
 dialog.accelerators.accelerator=Accelerator
 dialog.accelerators.action=Action
 dialog.accelerators.title=Accelerator map
-dialog.workspace_selector.preferences=Preferences
-dialog.workspace_selector.clone=Clone remote...
-dialog.workspace_selector.donate=Donate
-dialog.workspace_selector.import=Import from file...
 dialog.workspace_selector.new_workspace=New workspace...
 dialog.workspace_selector.open_workspace=Open workspace...
+dialog.workspace_selector.import=Import from file...
+dialog.workspace_selector.clone=Clone remote...
+dialog.workspace_selector.clone.setup_failed=<html>Failed to setup remote workspace. Reason:<br><b>{0}
+dialog.workspace_selector.clone.setup_failed.title=Remote workspace setup failed
+dialog.workspace_selector.vcs_info=<html>Please enter the URL of the repository/remote workspace you would like to clone below. In order to be able<br>\
+  to sync back to the remote workspace, you need to be have proper permissions on the remote repository for<br>\
+  the Git account you will enter here. Note that Git account is not your MCreator account.<br>
+dialog.workspace_selector.no_workspaces_loaded=<html><center>Failed to load recent workspaces<br>shortcuts. Re-open them to add<br>them to the list again.
+dialog.workspace_selector.no_workspaces=No recent workspaces
+dialog.workspace_selector.version=  Version {0}
```

Comment:
Trailing space will be removed here, so move space either back to code, or add \ at start

Reply:
:heavy_check_mark: Fixed